network.bisq.Bisq.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Should be contributed upstream.

Reply:
I don't see any Java projects on flathub pushing desktop entry or metadata to the upstream. e.g. https://github.com/flathub/net.runelite.RuneLite/blob/00301154d7163f50eb2fc3de54028785af07bb1d/net.runelite.RuneLite.json

For https://github.com/flathub/io.github.torbuntu.Leikr/blob/20da4419f20baab4a44899043f1b5f214fc43867/io.github.torbuntu.Leikr.json#L32, it naively push it to the upstream's assets directory, which IMO is not very useful, since the upstream doesn't really use it.

Reply:
maybe what you see is them not accepting.

`.desktop` is a basic requirement to be a "desktop application" on linux (and *BSD). It is what is being used to add the application to the various launching mechanisms from the various desktop environment (it's cross-desktop).

Reply:
> .desktop is a basic requirement to be a "desktop application" on linux (and *BSD).

Agreed. Java GUI applications also generates a `.desktop` file. But since I'm not an expert on Java GUI programs, what I can do is to find examples on flathub, but in vain. Can you point me to any such precedence? Otherwise, I think it's fair to suspect that generating .desktop entries not standardized in Java GUI programs (e.g. in Gradle).

Reply:
> maybe what you see is them not accepting.

For https://github.com/flathub/io.github.torbuntu.Leikr/blob/20da4419f20baab4a44899043f1b5f214fc43867/io.github.torbuntu.Leikr.json#L32, the upstream definitely accepted those files, since they simply lay in the upstream's repo. But they are not in use in a meaningful way (probably the gradle generated `.desktop` is the one placed in users' system)